src/library/trackset/baseplaylistfeature.h

```diff
@@ -79,22 +80,21 @@ class BasePlaylistFeature : public BaseTrackSetFeature {
     QModelIndex indexFromPlaylistId(int playlistId);
 
     PlaylistDAO& m_playlistDao;
-    TreeItemModel m_childModel;
     QModelIndex m_lastRightClickedIndex;
 
-    QAction* m_pCreatePlaylistAction;
-    QAction* m_pDeletePlaylistAction;
-    QAction* m_pAddToAutoDJAction;
-    QAction* m_pAddToAutoDJTopAction;
-    QAction* m_pAddToAutoDJReplaceAction;
-    QAction* m_pRenamePlaylistAction;
-    QAction* m_pLockPlaylistAction;
-    QAction* m_pImportPlaylistAction;
-    QAction* m_pCreateImportPlaylistAction;
-    QAction* m_pExportPlaylistAction;
-    QAction* m_pExportTrackFilesAction;
-    QAction* m_pDuplicatePlaylistAction;
-    QAction* m_pAnalyzePlaylistAction;
+    QAction* m_pCreatePlaylistAction{};
```

Comment:
We don't initialize variables in the header file.
This is unexpected in the Mixxx code base and  introduces a static hidden constructor, we like to avoid.

Reply:
If you like you may use parented_ptr to document the ownership.

Reply:
oh, I think that was an accident ^^